src/styles/blocks/menu.scss

```diff
@@ -0,0 +1,95 @@
+.menu {
```

Comment:
opening and closing icons are at different distances

![Screenshot_2](https://user-images.githubusercontent.com/89162624/208905403-d541cb0a-001f-4d4f-ac5a-25d4b1ad57a2.png)
![Screenshot_1](https://user-images.githubusercontent.com/89162624/208905405-c2a8aeb7-dca9-4998-a71e-0a21ccd6d52a.png)